package.json

```diff
@@ -2,7 +2,7 @@
     "private": true,
     "license": "EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0",
     "scripts": {
-        "prepare": "yarn build:extensions && yarn bundle:extensions",
+        "prepare:extensions": "yarn build:extensions && yarn bundle:extensions",
```

Comment:
I think we can simplify a bit here. Why not have `build:extensions` also call `yarn bundle:extensions` and get rid of `prepare:extensions`? 

```diff
diff --git a/package.json b/package.json
index 6acf6d4..5c05240 100644
--- a/package.json
+++ b/package.json
@@ -2,8 +2,7 @@
     "private": true,
     "license": "EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0",
     "scripts": {
-        "prepare:extensions": "yarn build:extensions && yarn bundle:extensions",
-        "build:extensions": "yarn --cwd vscode && yarn compile:extensions",
+        "build:extensions": "yarn --cwd vscode && yarn compile:extensions && yarn bundle:extensions",
         "compile:extensions": "cross-env NODE_OPTIONS=--max-old-space-size=8192 node ./src/compile.js",
         "bundle:extensions": "cross-env NODE_OPTIONS=--max-old-space-size=8192 node ./src/bundle.js",
         "publish:vsix": "node ./src/publish-vsix.js",
@@ -35,4 +34,4 @@
     "workspaces": [
         "vscode-builtin-extensions"
     ]
-}
\ No newline at end of file
+}
```

Reply:
It's better this way.